src/components/sessions/SessionControl.tsx

```diff
@@ -61,14 +61,14 @@ const SessionControl = () => {
     handleDismissModalPress();
   };
 
-  const onCancelSession = () => {
-    setSessionState("CANCEL");
-    setTimerState("STOPPED");
-    setTimeInSeconds(-1);
-    handleDismissModalPress();
-    // TODO: start real time tracking
-    // TODO: discard session data
-  };
+  // const onCancelSession = () => {
+  //   setSessionState("CANCEL");
+  //   setTimerState("STOPPED");
+  //   setTimeInSeconds(-1);
+  //   handleDismissModalPress();
+  //   // TODO: start real time tracking
+  //   // TODO: discard session data
+  // };
```

Comment:
is this meant to be commented out?

Reply:
I'm not sure, the design doesn't have a close button, but there was an X button using this function.
I've removed the button and commented out this piece of code to leave as reference in case is needed elsewhere


<img width="616" alt="Screenshot 2024-06-28 at 5 03 49 PM" src="https://github.com/LangaraBeetles/weaup-react-native-app/assets/57273227/b6d31e82-15a6-4cff-bd19-84ecffb6f3e9">

Reply:
Ok fair enough!